### ISSUE TITLE
v0.2.0 Failed creating cluster:  [WorkerRootVolumeSize, ControllerRootVolumeSize] do not exist in the template

### ISSUE THREAD

**Issue by nelsonenzo** [coreos/coreos-kubernetes#154]
after building 0.2.0 and trying to launch a cluster, I am seeing the following error:
```
Failed creating cluster: ValidationError: Parameters: [WorkerRootVolumeSize, ControllerRootVolumeSize] do not exist in the template
```
my cluster.yaml file:

```
# Unique name of Kubernetes cluster. In order to deploy
# more than one cluster into the same AWS account, this
# name must not conflict with an existing cluster.
clusterName: kubernetes

# Name of the SSH keypair already loaded into the AWS
# account being used to deploy this cluster.
keyName: ec2keyname

# Region to provision Kubernetes cluster
region: us-east-1

# Availability Zone to provision Kubernetes cluster
#availabilityZone:

# DNS name routable to the Kubernetes controller nodes
# from worker nodes and external clients. The deployer
# is responsible for making this name routable
externalDNSName: kubernetes.development.company.com

# Disk size (GB) for controller node
controllerRootVolumeSize: 100

# Number of worker nodes to create
workerCount: 2

# Disk size (GB) for worker nodes
workerRootVolumeSize: 100

```

**Comment by aaronlevy** [comment]
I wasn't able to reproduce the issue by uncommenting / modifying the controller/worker root volume sizes. Can you maybe try with the release binary from https://github.com/coreos/coreos-kubernetes/releases/tag/v0.2.0 ?

**Comment by nelsonenzo** [comment]
Thank you @aaronlevy , the release binary did work.  These are the steps I took to build it. If I could get kube-aws to work from builds I would be happy to test future pull requests, at the very least.

```
export GOPATH=/Users/nhernandez/Documents/dev/golang
cd /Users/nhernandez/dev/golang/src/github.com/coreos
git clone https://github.com/coreos/coreos-kubernetes.git
cd coreos-kubernetes/multi-node/aws
git checkout v0.2.0
./build
cd bin
./kube-aws up

Initialized TLS infrastructure
Wrote kubeconfig to /Users/nhernandez/Documents/dev/golang/src/github.com/coreos/coreos-kubernetes/multi-node/aws/bin/clusters/kubernetes/kubeconfig
Waiting for cluster creation...
Failed creating cluster: ValidationError: Parameters: [WorkerRootVolumeSize, ControllerRootVolumeSize] do not exist in the template
	status code: 400, request id: febee4a7-8e4c-11e5-ab94-97649380cde1

```

**Comment by ankitforcode** [comment]
The issue is reproducible at my end.

Failed creating cluster: ValidationError: Parameters: [WorkerRootVolumeSize, ControllerRootVolumeSize] do not exist in the template

**Comment by nelsonenzo** [comment]
@ankitforcode Were you able to reproduce it using the release binary  https://github.com/coreos/coreos-kubernetes/releases/tag/v0.2.0, or the via a custom build?

Using my custom build, I have at least identified what was causing the problem using ```kube-aws render```
My custom build is generating a template with extra references to ControllerRootVolumeSize and WorkerRootVolumeSize, as seen here in this diff between custom build render and release binary render:
```
81,85d80
<         "ControllerRootVolumeSize": {
<             "Default": "30",
<             "Description": "Controller root volume size (GiB)",
<             "Type": "String"
<         },
115,119d109
<         },
<         "WorkerRootVolumeSize": {
<             "Default": "30",
<             "Description": "Worker root volume size (GiB)",
<             "Type": "String"
356,365d345
<                 "BlockDeviceMappings": [
<                     {
<                         "DeviceName": "/dev/xvda",
<                         "Ebs": {
<                             "VolumeSize": {
<                                 "Ref": "ControllerRootVolumeSize"
<                             }
<                         }
<                     }
<                 ],
462,471d441
<                 "BlockDeviceMappings": [
<                     {
<                         "DeviceName": "/dev/xvda",
<                         "Ebs": {
<                             "VolumeSize": {
<                                 "Ref": "WorkerRootVolumeSize"
<                             }
<                         }
<                     }
<                 ],
```

**Comment by dhoeric** [comment]
This can be solved in 2 ways:
1) Disable all "*RootVolumeSize" in *.go and re-build the ./bin/kube-aws
2) Render a new template.json and upload to your own server

**Comment by bcwaldon** [comment]
Just released [v0.3.0](https://github.com/coreos/coreos-kubernetes/releases/tag/v0.3.0), which addresses this issue.
